bootstrap_gen.py

```diff
@@ -0,0 +1,138 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
@andrew-matteson this file should be committed [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap_gen.py). Apologies for the confusion, improving this is being actively worked on.